src/RA_Dlg_GameLibrary.cpp

```diff
@@ -16,13 +16,15 @@
 #include "services\IConfiguration.hh"
 #include "services\ServiceLocator.hh"
 
-#define KEYDOWN(vkCode) ((GetAsyncKeyState(vkCode) & 0x8000) ? true : false)
+_NODISCARD _CONSTANT_FN
+KeyDown(_In_ int vkCode) noexcept { return ((GetAsyncKeyState(vkCode) & 0x8000) ? true : false); }
 
-namespace {
+
+namespace ra {
 
 const char* COL_TITLE[] = { "ID", "Game Title", "Completion", "File Path" };
 const int COL_SIZE[] = { 30, 230, 110, 170 };
-static_assert(SIZEOF_ARRAY(COL_TITLE) == SIZEOF_ARRAY(COL_SIZE), "Must match!");
+static_assert(ra::SizeOfArray(COL_TITLE) == ra::SizeOfArray(COL_SIZE), "Must match!");
 const bool bCancelScan = false;
 
 std::mutex mtx;
```

Comment:
Kept on getting intellisense errors about `mtx `being ambiguous. Unnamed namespace should be avoid anyway, it also sounds like an oxymoron.